app/services/redis.py

```
@@ -0,0 +1,55 @@
+import redis.asyncio as aioredis
+import json
+from typing import Optional, Dict, Any
+from app.core.config import settings
+
+redis_client: Optional[aioredis.Redis] = None
+
+
+async def get_redis() -> aioredis.Redis:
+    """Get Redis client instance."""
+    global redis_client
```

Pylint (3.3.7)

[warning] Using the global statement

(W0603)